include/arch/arc/v2/linker-mwdt.ld

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
It seems that `linker-gnu.ld` and `linker-mwdt.ld` does have much in common.
We should try to see if they could be aligned, but it can go in for now.

Reply:
Yep, I'd better keep it as a separate enhancement.

Reply:
agreed. it's for later improvement